lib/lib.go

```diff
@@ -255,8 +255,8 @@ func (p *Podium) buildGetCountURL(leaderboard string) string {
 	return p.buildURL(pathname)
 }
 
-func (p *Podium) buildGetMembersAroundMemberURL(leaderboard, memberID string, pageSize int) string {
-	pathname := fmt.Sprintf("/l/%s/members/%s/around?pageSize=%d", leaderboard, memberID, pageSize)
+func (p *Podium) buildGetMembersAroundMemberURL(leaderboard, memberID string, pageSize int, lastIfNotFound bool, order string) string {
+	pathname := fmt.Sprintf("/l/%s/members/%s/around?pageSize=%d&getLastIfNotFound=%t&order=%s", leaderboard, memberID, pageSize, lastIfNotFound, order)
```

Comment:
shouldn't the querystring be "lastIfNotFound" to be consistent?

Reply:
also, these 2 new qss are already supported in the original code? Because all I can see is the url being built, but the new values aren't handled.

Reply:
The API already describes these query parameters, I merely added them to the go library.
I could change the bool method parameter to 'getLastIfNotFound' if that would be better.

Reply:
got it! I would change it to have the same names

Reply:
Done!